src/AtvService.php

```diff
@@ -559,6 +559,38 @@ public function getDocument(string $id, bool $refetch = FALSE): AtvDocument {
 
   }
 
+  /**
+   * Check if documents are found with given Trasaction ID.
+   *
+   * @param string $id
+   *   Transaction id.
+   *
+   * @return bool|null
+   *   Boolean if found.
+   *
+   * @throws \Drupal\helfi_atv\AtvDocumentNotFoundException
+   * @throws \Drupal\helfi_atv\AtvFailedToConnectException
+   * @throws \GuzzleHttp\Exception\GuzzleException|\Drupal\helfi_helsinki_profiili\TokenExpiredException
+   */
+  public function checkDocumentExistsByTransactionId(string $id) {
```

Comment:
Hmm. Tää siis tsekkaa onks transaction id olemassa, miks palauttaa NULL / boolean?

Mitä eroo false / null paluuarvoilla?

Reply:
Joo, toi NULL palautuu vaan jos responsessa ei oo results mukana, varmaan meillä olis kyl isompi ongelma jos näin on, niin toi NULLIN palautus voipi olla vähän turhahko

Reply:
![image](https://github.com/City-of-Helsinki/drupal-module-helfi-atv/assets/892920/b741ec8a-57ab-4849-91f5-f6b5cca7abf9)

Reply:
> Joo, toi NULL palautuu vaan jos responsessa ei oo results mukana, varmaan meillä olis kyl isompi ongelma jos näin on, niin toi NULLIN palautus voipi olla vähän turhahko

joo vois ihan hyvin heittää virheen jos sitä ei oo, ja aina palauttelee true/false